src/services/httpsRequestUtils.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { makeHttpsRequest, base64Encode } from './httpsRequestUtils'
+import { exampleEventBody } from '../testUtils/events/exampleEventBody'
+
+function getSecrets() {
+  return {
+    ZENDESK_USER_ID: process.env.ZENDESK_USER_ID ?? '', // TODO grab from AWS Secrets Manager
```

Comment:
This stuff will need to come from the secrets retrieval code I've now merged into the main branch, should hopefully work without any messing about!

Reply:
now properly accessing secrets from AWS following your merge